### ISSUE TITLE
Feature Request: Disable Swipe Tutorial When Using a Mouse

### ISSUE THREAD

**Issue by ahmadalsaady** [ubports/ubuntu-touch#1075]
- Device: bacon
- Channel: release candidate
- Build: 2019-W11

Could you add a feature to disable "swipe from the edge tutorial" when a mouse is connected and touch is broken or disabled as it keeps appearing as soon as the mouse cursor stops moving

**Comment by dobey** [comment]
Is this with a USB mouse? Did you plug it in while installing UT? The tutorial should only run on first boot after installing UT (or in future maybe immediately after an upgrade). It sounds like the tutorial ran much later for you.

**Comment by ahmadalsaady** [comment]
Yes it is a USB mouse, my device has a broken touch so I used the mouse on  a fresh install and this message started to appear directly after I finished the setup process and it just won't go away as I can't simulate the swipes using the mouse
